pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+
+[project]
+name = "flexgen"
+version = "1.0"
+description = "Running large language models like OPT-175B/GPT-3 on a single GPU. Focusing on high-throughput large-batch generation."
+readme = "README.md"
+requires-python = ">=3.7"
+classifiers = [
```

Comment:
```suggestion
install_requires = ["numpy", "tqdm", "pulp", "attrs", "transformers>=4.24", "torch>=1.12"]
classifiers = [
```

Reply:
It leads to 
```
 ValueError: invalid pyproject.toml config: `project`.
      configuration error: `project` must not contain {'install_requires'} properties
      [end of output]
  ```

Reply:
the proper name is `dependencies`, updated